Classes Chapter (#7)
LaunchCodeEducation/csharp-web-dev-curriculum: content/classes/_index.md

---
title: "Classes"
date: 2023-01-23T13:36:52-06:00
draft: false
weight: 4
originalAuthor: <no value> # to be set by page creator
originalAuthorGitHub: <no value> # to be set by page creator
reviewer: # to be set by the page reviewer
reviewerGitHub: # to be set by the page reviewer
lastEditor: # update any time edits are made after review
lastEditorGitHub: # update any time edits are made after review
lastMod: # UPDATE ANY TIME CHANGES ARE MADE
---

## Learning Objectives

> After completing all of the work in this lesson, you should be able to:

Reviewer: ```suggestion
After completing this chapter, you should be able to do the following:
```

1. Use access modifiers to customize fields and methods.
1. Override built-in methods such as `ToString` and `Equals`.
1. Understand equality and sameness in C#.

## Key Terminology

Throughout this lesson you should make note of the following terms and their definitions:

1. class
1. object
1. fields
1. methods
1. members
1. instance
1. `this` keyword
1. shadowing
1. access level
1. access modifier
1. `public`
1. `private`
1. default access
1. assembly
1. world-level
1. encapsulation
1. accessor
1. getter
1. setter
1. auto-implemented property
1. backing field
1. default constructor

> 1. instance method

Reviewer: Please update format.

Author: Done!
